src/controllers/StudentController.js

```diff
@@ -0,0 +1,56 @@
+const Student = require('../models').Student;
+
+module.exports = {
+  async getAll(req, res) {
+    try {
+      const students = await Student.findAll();
+      return res.status(200).send(students);
+    } catch (error) {
+      console.log(error);
+    }
```

Comment:
Нужно вернуть ответ, иначе зависнет запрос в ожидании

```suggestion
    } catch (error) {
      console.log(error);
    res.sendstatus(500)
    }
```

Reply:
ок, исправлено